PlottingAnalysis/TS_TopFeatures.m

```diff
@@ -125,10 +147,12 @@
     classNumbers = arrayfun(@(x)sum(TimeSeries.Group==x),1:numClasses);
     isBalanced = all(classNumbers==classNumbers(1));
     if isBalanced
-        fn_testStat = GiveMeFunctionHandle(whatTestStat,numClasses,'acc',0);
+        fn_testStat = @(XTrain,yTrain,XTest,yTest,numFolds) GiveMeCfn(whatTestStat,...
+                            XTrain,yTrain,XTest,yTest,numClasses,0,'acc',[],numFolds);
         fprintf(1,'Using overall classification accuracy as output measure\n');
     else
-        fn_testStat = GiveMeFunctionHandle(whatTestStat,numClasses,'balancedAcc',1);
+        fn_testStat = @(XTrain,yTrain,XTest,yTest,numFolds) GiveMeCfn(whatTestStat,...
+                            XTrain,yTrain,XTest,yTest,numClasses,0,'balancedAcc',[],numFolds);
```

Comment:
In this and throughout, why does `numFolds` need to be an input to the inline function? Can't it just be fixed at the user's value? Then you don't need a fifth input to the inline functions defined later...?

Reply:
So that we can specific 0 folds later, which gives the model, rather than the CV model (which can't be used for prediction)